src/Core/Components/AppBar/FluentAppBar.razor.cs

```diff
@@ -39,16 +39,19 @@ public partial class FluentAppBar : FluentComponentBase
     [Parameter]
     public EventCallback<bool> PopoverVisibilityChanged { get; set; }
 
+    [Parameter]
+    public IEnumerable<IAppBarItem>? Apps { get; set; }
```

Comment:
Could you add a comment for the documentation?

I think that `Items` could be a better name. What is your opinion?

Reply:
Yes, was doubting the name as well. Will change to Items